npm/create-cypress-tests/package.json

```diff
@@ -5,9 +5,10 @@
   "private": false,
   "main": "index.js",
   "scripts": {
-    "build": "yarn prepare-example && tsc -p ./tsconfig.json && node scripts/example copy-to ./dist/initial-template && yarn copy \"./src/**/*.template.js\" \"./dist/src\"",
```

Comment:
Hmm, there has to be an easier way, right? Why is `copy-templates` necessary? Will something like [this](https://github.com/jonschlinkert/copy#api) work?

Reply:
just try build the scope